app/view_objects/results_view.rb

```diff
@@ -258,6 +258,14 @@ def placement_schools_summary(course)
     end
   end
 
+  def country
+    query_parameters['c']
+  end
+
+  def devolved_nation?
```

Comment:
the `DEVOLVED_NATIONS` constant could then re reused here

NameofModel::DEVOLVED_NATIONS.include(country)

Reply:
Done